sensor_msgs/JointState state
int32 seq_num
bool finished
bool no_check_forbidden_area
